include/net/fota_download.rst

```diff
@@ -40,7 +28,19 @@ To use HTTPS instead, apply the changes described in :ref:`the HTTPS section of
 
 The FOTA download library is used in the :ref:`http_application_update_sample` sample.
 
+.. note::
+   To upgrade the upgradable bootloader, you must provide two firmware files: one linked against the S0 partition and one linked against the S1 partition.
+   See :ref:`upgradable_bootloader` for more information.
+
+   To do this, you must provide two paths in the ``file`` argument, separated by a space character:
 
+   *.The first entry must point to the MCUboot variant linked against the S0 partition.
```

Comment:
Should the "." in the start of the sentence be there?

Reply:
Fixed